packages/stark-ui/src/modules/table/components/column.component.html

```diff
@@ -4,7 +4,7 @@
 	<th mat-header-cell *matHeaderCellDef [ngClass]="getHeaderClassNames()">
 		<div class="header-cell-content">
 			<div [ngSwitch]="sortDirection" class="sort-header" (click)="onSortChange()">
-				<span>{{ getHeaderLabel() }}</span>
+				<span>{{ getHeaderLabel() | translate }}</span>
```

Comment:
Can you move the translation logic to the `getHeaderLabel` inside the Component class instead? That way it is more flexible in case we need to add more logic to what needs to be rendered